samples/csharp/MicrosoftEduImportFromGoogle/README.md

```diff
@@ -0,0 +1,85 @@
+# Microsoft EDU import from Google Classroom
```

Comment:
We should explicitly mention that we need to create an app registration on both Google and MSFT Side. Also it would be great to have screenshots of the app registration, where to find the client id, client secret etc. page on both sides

Reply:
added prerequisites for google and microsoft, there is info from official sources in both sides, I just added links for reference on each topic.